src/api/users/src/services/firestore.js

```diff
@@ -33,7 +33,7 @@ if (
       type: FIREBASE_TYPE,
       project_id: FIREBASE_PROJECT_ID,
       private_key_id: FIREBASE_PRIVATE_KEY_ID,
-      private_key: FIREBASE_PRIVATE_KEY,
+      private_key: FIREBASE_PRIVATE_KEY.replace(/\\n/g, '\n'), // escape the newline characters so docker can properly parse the key
```

Comment:
Why are there newline characters in it at all?  It should be a single line of text.

Reply:
It is a single line of text but it does have newline characters in it. I was getting private key errors without this fix ([that someone suggested on stackoverflow](https://stackoverflow.com/a/41044630/13809136))

Reply:
In other files where we do this, we've been reading a file (cc @manekenpix, who has done this a bunch and probably has advice).